internal_docs/review-apps.md

```diff
@@ -0,0 +1,26 @@
+# Pull Request Review apps
+
+This repo uses [Heroku review apps](https://devcenter.heroku.com/articles/github-integration-review-apps)
+to give reviewers a way to easily preview and test changes to the app.
+
+If a pull request is created from a fork of this repo (typical if the
+contributor is not part of the alphagov organisation) then the review app will
+not be created automatically; however it can be created manually from the
+Heroku dashboard.
+
+Previews of Pull Requests are automatically published to a URL which has the
+prefix `govuk-prototype-kit` followed by the identifier number of the pull request.
+
+For example, pull request #137 would be deployed to `govuk-prototype-kit-pr-137.herokuapp.com`.
+
+The govuk-design-system-ci should create a GitHub deployment as Heroku builds
```

Comment:
Noob question: should govuk-design-system-ci display inside back quotes?

Reply:
Ooh err I struggle with this one; I personally find it can get quite busy putting terms in backquotes every time so tend to take an backquote the first time approach, but I'm not consistent.

Also here govuk-design-system-ci is a user so it could be @govuk-design-system-ci 

And I'm not sure why I've put "The" haha 🤦